### ISSUE TITLE
wrong messurement SDS011

### ISSUE THREAD

**Issue by dankyn** [opendata-stuttgart/sensors-software#834]
The sensor SDS011 delivered incorrect measured values very hight, while using its software the parameters are correct.
The only thing i noticed that the esp has the different pin gpio position
https://postimg.cc/Fk3DwPVX
airRohr: NRZ-2020-132-B3/IT
mounting FS...
opened config file...
parsed json...
output debug text to displays...
Connecting to Ospiti_ng
.......
WiFi connected, IP is: 192.168.1.129
Starting Webserver... 192.168.1.129

ChipId: 5811203

MAC Id: ecfabc58ac03
Read SDS...: 
Stopping SDS011...
Send to :
sensor.community
Madavi.de
----
Auto-Update active...
SNTP synced: Wed Nov 25 08:21:46 2020

twoStageOTAUpdate
output debug text to displays...
PM10:  294.93
PM2.5: 272.83

**Comment by ricki-z** [comment]
This is a duplicate of #835. Please post your issues only once!
